skpro/distributions/uniform.py

```diff
@@ -0,0 +1,134 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Uniform probability distribution."""
+
+__author__ = ["an20805"]
+
+import numpy as np
+import pandas as pd
+
+from sktime.proba.base import BaseDistribution
+
+
+class Uniform(BaseDistribution):
+    """Continuous uniform distribution.
+
+    Parameters
+    ----------
+    lower : float
+        Lower bound of the distribution.
+    upper : float, must be greater than lower
+        Upper bound of the distribution.
+    index : pd.Index, optional, default = RangeIndex
+    columns : pd.Index, optional, default = RangeIndex
+
+    Example
+    -------
+    >>> from sktime.proba import Uniform
+
+    >>> u = Uniform(lower=0, upper=5)
+    """
+
+    _tags = {
+        "authors": ["an20805"],  
+        "capabilities:approx": ["pdfnorm"],
+        "capabilities:exact": ["pdf", "log_pdf", "cdf", "ppf", "mean", "var"],
+        "distr:measuretype": "continuous",
+    }
+
+    def __init__(self, lower, upper, index=None, columns=None):
+        if lower >= upper:
+            raise ValueError("Upper bound must be greater than lower bound.")
```

Comment:
```suggestion
            raise ValueError(f"Upper bound ({upper}) must be greater than lower bound {lower}.")
```
in this way the error message gives more information about the problem